### DISCUSSION TITLE
GitHub Sponsors: Within two weeks?

### DISCUSSION THREAD

**Discussion by jgneff** [community/community#21798] [New to GitHub]
<p>I applied to GitHub Sponsors on February 3, 2020, which is almost six weeks ago, yet my wait list page still says, “Within two weeks, you’ll get an email with instructions on setting up your GitHub Sponsors profile.” My bank account country is Canada. Is there something holding up my application, or is it just taking GitHub much longer than expected? Thank you for making the sponsorship program available!</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/jgneff">@jgneff</a>, thanks for being here! Given that GitHub Sponsors is no longer in Beta and it’s been 6 weeks, I suggest reaching out to GitHub Support directly to resolve this issue. Good luck! </p>

**Comment by jgneff** [comment]
<p>Thanks, <a class="mention" href="/logankilpatrick">@logankilpatrick</a>. I got the “Welcome to GitHub Sponsors” message exactly two hours after your reply, so it seems it was enough just to post my question here. Now I’m having other related troubles, which I may post as separate questions. Thanks again for your help!</p>
